example/zrc7/mint.js

```diff
@@ -0,0 +1,70 @@
+const {BN, Long, bytes, units} = require('@zilliqa-js/util');
+const {Zilliqa} = require('@zilliqa-js/zilliqa');
+const {
+    toBech32Address,
+    getAddressFromPrivateKey,
+} = require('@zilliqa-js/crypto');
+const fs = require('fs');
+
+
+async function main() {
+    const zilliqa = new Zilliqa('https://dev-api.zilliqa.com');
+    const CHAIN_ID = 333;
+    const MSG_VERSION = 1;
+    const VERSION = bytes.pack(CHAIN_ID, MSG_VERSION);
+    privkey = '07e0b1d1870a0ba1b60311323cb9c198d6f6193b2219381c189afab3f5ac41a9';
+    zilliqa.wallet.addByPrivateKey(
+        privkey
+    );
+    const address = getAddressFromPrivateKey(privkey);
+    console.log("Your account address is:");
+    console.log(`${address}`);
+    const myGasPrice = units.toQa('2000', units.Units.Li); // Gas Price that will be used by all transactions
+
+    const metadata = JSON.parse(fs.readFileSync('./zrc-7-metadata.json'))
```

Comment:
First, I think this is a bad practice.  We don't want to store the all metadata in on-chain.
Second, ZRC-7 is about the metadata structure. Therefore, this deployment is irrelevant.
Remove This File.